ultravox/tools/tts.py

```diff
@@ -0,0 +1,48 @@
+import io
+import os
+from typing import Optional
+from xml.sax import saxutils
+
+import numpy as np
+import requests
+import soundfile as sf
+
+
+def _make_ssml(voice: str, text: str):
+    return f"""
+    <speak version="1.0" xml:lang="en-US">
+        <voice xml:lang="en-US" name="{voice}">
+            {saxutils.escape(text)}
+        </voice>
+    </speak>"""
+
+
+class AzureTts:
+    DEFAULT_VOICE = "en-US-JennyNeural"
```

Comment:
Is there a way to randomize this or cycle through multiple voices?

Reply:
Not at present. We could allow -V to accept either a string or a list though in a follow up PR.